.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
This is the devcontainer config I used for this PR.

Optionally, I can move this devcontainer stuff to a separate dedicated PR.